views/content/edit.php

```diff
@@ -107,6 +107,93 @@ class="input-mini timeentry editable" type="text"
         </select>
       </div>
     </div>
+      <?php
+      $problemTypeConfig = $config['problem_type'];
+      if (isset($problemTypeConfig) ) {
+          $problemTypeTitle = $problemTypeConfig['title'];
+          $problemTypeList  = $problemTypeConfig['types'];
+          ?>
+          <div class="control-group">
+              <label class="control-label" for="problem_type"><?php echo $problemTypeTitle;?></label>
+              <div class="controls">
+                  <select id="problem_type" name="problem_type" class="input-large editable" disabled>
+                      <option value="">--SELECT--</option>
+                      <?php
+                      foreach ($problemTypeList as $problemType) {
+                          $selected = ($problem_type == $problemType) ? " selected " : "";
+                          ?>
+                          <option value="<?php echo $problemType; ?>" <?php echo $selected;?>><?php echo $problemType; ?></option>
+                          <?php
+                      }
+                      ?>
+                  </select>
+              </div>
+          </div>
+          <?php
+      }
+      ?>
+      <div class="control-group">
+          <label class="control-label" for="subsystem">Subsystem</label>
+          <div class="controls">
+              <input type="text" placeholder="Subsystem" id="subsystem" name="subsystem" value="<?php echo $subsystem;?>" class="input-xlarge editable" disabled>
+          </div>
+      </div>
+      <div class="control-group">
+          <label class="control-label" for="owner_team">Owner team</label>
+          <div class="controls">
+              <input type="text" placeholder="Owner Team" id="owner_team" name="owner_team" value="<?php echo $owner_team;?>" class="input-xlarge editable" disabled>
+          </div>
+      </div>
+      <?php
+
+      $impactTypeConfig = $config['impact_type'];
+      if (isset($impactTypeConfig) ) {
```

Comment:
Again, would do `array_key_exists` instead

Reply:
I will make this change at all suggested places.